docs/collecting-data/collecting-from-own-applications/mobile-trackers/migration-guide/index.md

```diff
@@ -0,0 +1,229 @@
+---
+title: "Migration guide"
+date: "2022-08-30"
+sidebar_position: 1000
+---
+
+```mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+```
+
+# Migration guide from version 3.x to 4.0
```

Comment:
Do the benefits of having this as a single page justify having more redirects? I would keep as is...

Reply:
It was meant to be more in line with some other tracker docs but not that important, changed it to have separate migration pages as before.